package.json

```
@@ -136,6 +137,10 @@
           "command": "vscode-objectscript.compile",
           "when": "editorLangId =~ /^objectscript/ && vscode-objectscript.connectActive"
         },
+        {
+          "command": "vscode-objectscript.refreshLocalFile",
+          "when": "editorLangId =~ /^objectscript/ && vscode-objectscript.connectActive"
```

```suggestion
          "when": "resourceScheme == file && editorLangId =~ /^objectscript/ && vscode-objectscript.connectActive"
```
I don't think the command should be offered for an isfs document.